README.md

```diff
@@ -27,13 +27,30 @@ npm install
 
 # copy default environment config
 cp env.sample .env
+```
+
+You will then also need to create a database and set up the table definitions.
+
+If you have installed PostgreSQL in a way that works without requiring a username and password, use:
+
+```
+# create a database; -U and -W are only necessary if your pg instance needs authentication
+createdb webmaker
```

Comment:
do we want to show `-U` in this instruction too, since it uses the format `-U username`